src/main/java/com/google/sps/utils/data/AnimalHistory.java

```diff
@@ -0,0 +1,71 @@
+package com.google.sps.data;
+
+public class AnimalHistory {
+    private int year;
+    private String code;
+    private String category;
+
```

Comment:
What are the code and category parameters?

Reply:
I got those categories from here: https://apiv3.iucnredlist.org/api/v3/species/history/id/12392?token=9bb4facb6d23f48efbf424bb05c0c1ef1cf6f468393bc745d42179ac4aca5fee